api-gateway/src/main/resources/application.yaml

```diff
@@ -47,9 +47,13 @@ logging:
     root: INFO
     org.springframework.cloud.gateway.route.RouteDefinitionLocator: INFO
     org.springframework.cloud.gateway: TRACE
+#  pattern.level: "trace_id=%mdc{traceId} span_id=%mdc{spanId} trace_flags=%mdc{traceFlags} %p"
```

Comment:
There is an open issue to show traceId and spanId for webflux https://github.com/spring-projects/spring-boot/issues/33372